src/issue/screens/issue.screen.js

```diff
@@ -160,10 +159,11 @@ class Issue extends Component {
         issueURLParam ? issueCommentsURL : issueParam.comments_url
       ),
     ]).then(() => {
+      const issue = this.props.issue;
```

Comment:
@chinesedfan can as earlier receive variable `issue` (144 line)? Then you just do not need this line.

Reply:
@lex111 No, we should read after those 2 promises, `getIssueFromUrlByDispatch` and `getIssueCommentsByDispatch`, are fulfilled. Or `this.props.issue` may refer to the previous wrong issue.

That's what I mean "the fetched issue". Sorry for my poor English pr description.

Reply:
Nope that shouldn't be the case :)

In the top of the file, we're mapping parts of our state object into our props. So `issue` isn't a fixed object parameter here, but will always reflect what's happening with state. If `getIssueFromUrlByDispatch` and/or `getIssueCommentsByDispatch` update `state.issue` where state is our Redux container - our issue props will always reflect this. So we should be good with leaving it as part of the destructuring in line 144 😃 

Please let me know to clarify this if it's still not making any sense to you mate

Reply:
Right. `this.props.issue` will always be updated by redux, but in line 144, `issue` is just a temporary local variable destructed from `this.props`, which will never be updated.

Reply:
Oh I'm sorry, you're right here. It gets modified in the promise call and our current local variable isn't really going to reflect this.

Cheers thank you mate, and I'm sorry for causing any confusion 😞